src/helix-ui/elements/HXErrorElement.js

```diff
@@ -1,21 +1,13 @@
 import { HXElement } from './HXElement';
 import shadowStyles from './HXErrorElement.less';
-
-const tagName = 'hx-error';
-const template = document.createElement('template');
-
-template.innerHTML = `
-  <style>${shadowStyles}</style>
-  <hx-icon type="exclamation-circle"></hx-icon>
-  <slot></slot>
-`;
+import shadowMarkup from './HXErrorElement.html';
 
 export class HXErrorElement extends HXElement {
     static get is () {
-        return tagName;
+        return 'hx-error';
     }
 
-    constructor () {
-        super(tagName, template);
+    static get template () {
+        return `<style>${shadowStyles}</style>${shadowMarkup}`;
```

Comment:
Because we're `import`ing the markup, it gets its whitespace collapsed before we use it, which results in:

1. a smaller string (thus smaller file size)
2. no misinterpreted whitespace

`'<style>' + shadowStyles$3 + '</style>' + shadowMarkup$3`